widget/entry.go

```diff
@@ -525,6 +532,12 @@ func (e *Entry) TappedSecondary(pe *fyne.PointEvent) {
 	clipboard := fyne.CurrentApp().Driver().AllWindows()[0].Clipboard()
 	super := e.super()
 
+	undoItem := fyne.NewMenuItem("Undo", func() {
```

Comment:
this can be inlined without the new `func()` - just pass `e.undo` as the action parameter.